examples/hello-world/Cargo.toml

```diff
@@ -5,5 +5,7 @@ edition = "2018"
 publish = false
 
 [dependencies]
-axum = { path = "../../axum" }
+axum = { path = "../../axum", features = ["headers"] }
+axum-macros = { path = "../../axum-macros" }
```

Comment:
Wouldn't it be better to have a separate macro / derive example and leave the hello-world example as-is?

Reply:
Yea I'm just using this for testing while developing.